libvips/foreign/vips2tiff.c

```diff
@@ -1039,13 +1041,13 @@ ready_to_write( Wtiff *wtiff )
 
 	/* "squash" float LAB down to LABQ.
 	 */
-	if( wtiff->squash &&
+	if( wtiff->bitdepth == 1 &&
```

Comment:
How about allowing 8 as well here? Setting bitdepth to 1 to save 32-bit float as 8-bit uint seems unintuitive.

Reply:
Ok then l will do it as follows: Allow bitdepth = 8 for CIE LAB but not for integral images because there the proper way to change bitdepth from e.g. 16 to 8 is to change the type

Reply:
Sounds good! You'll need to allow `1` as well, of course.

Reply:
Why do I have to allow `1` as well? CIE LAB allows for squash to 8 bit. I don't get your point.

Reply:
You set `bitdepth = 1` if squash is on, so you'll need to allow that here for lab -> labq squash to still work (I think).

Reply:
I see I did it differently, I set bitdepth to 8 in case squash is on.

Reply:
But won't that break the old behaviour of squash writing 8-bit uchar as 1-bit? I'm probably confused.

Reply:
No you are not. I'm the one who missed that and now I get your point. Thank you so much.